azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/features/VirtualMachineApi.java

```diff
@@ -131,5 +132,27 @@ VirtualMachine create(@PathParam("vmname") String vmname,
    @Path("/{name}/powerOff")
    void stop(@PathParam("name") String name);
 
+   /**
+    * Generalize the virtual machine
+    */
+   @Named("generalize")
+   @POST
+   @Path("/{name}/generalize")
+   @ResponseParser(StatusCodeParser.class)
```

Comment:
I think this is here because we need to generalize the VM before we can capture its image. @jtjk: what do you think about the recommended change?